packages/bottlerocket-licenses/bottlerocket-licenses.spec

```
@@ -0,0 +1,32 @@
+%global _cross_first_party 1
+%global debug_package %{nil}
+
+Name: %{_cross_os}bottlerocket-licenses
+Version: 0.0
+Release: 0%{?dist}
+Summary: bottlerocket-licenses
+License: Apache-2.0 OR MIT
+URL: https://github.com/bottlerocket-os/bottlerocket
+
+Source100: COPYRIGHT
+Source101: LICENSE-MIT
+Source102: LICENSE-APACHE
+
+%description
+%{summary}.
+
+%prep
+%setup -T -c
+
+%build
+
+%install
+mkdir -p %{buildroot}/usr/share/bottlerocket/
```

Looks like `bottlerocket` may be a mistake here since we're actually putting licenses in `/usr/share/licenses`?

In any case, this directory isn't quite right, `/usr/share/licenses` is mounted by a [systemd unit set up in `release`](https://github.com/bottlerocket-os/bottlerocket/blob/6a66bb2170c56a08dca4ac8ed279c78cc18017fa/packages/release/release.spec#L184)

```suggestion
mkdir -p %{buildroot}/%{_cross_licensedir}
```

You can generally just find/replace `/usr/share/licenses` with `%{_cross_licensedir}`.